dev/package-examples/longdocs-1.0.4/docs/README.md

```diff
@@ -11,6 +11,14 @@ This integration is considered to be in _beta_.
 
 This is a [link](https://github.com/elastic/integrations-registry) inside the docs.
 
+## Examples from a file
+
+The following file shows how an event might look like:
+
+```
+{{ file /docs/data.json }}
```

Comment:
@jfsiii Looking for input on this one. It's simpler then what I asked for in https://github.com/elastic/integrations-registry/pull/70 and could be enough to get started.

Reply:
Exactly. Will use this for now. One thing we probably will need in addition is:

```
{{ > {{prefix}}/docs/data.json }}
```
The path on your end will have a prefix and will need replacement I guess. So we have mustache in mustache :-) Suggestions?

Reply:
I went for now with `{{ > ./data.json }}`. The assumption is that all paths are relative to where they are specified.